### PR TITLE
Saving facing direction works now. Simplied Save Game Manager. Added …

### PR DESCRIPTION
**Pull request by kocosephia**
…Level Manager

### CODE REVIEW

File: WhosAGoodBoy/Who's A Good Boy/Assets/Scripts/GameFiles/LevelManager.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    public static LevelManager Instance;
+
+    void Awake()
+    {
+        //Check if instance already exists
+        if (Instance == null)
+            Instance = this;
+        else if (Instance != this)
+            Destroy(this);
```
Comment (nihakue) [inline WhosAGoodBoy/Who's A Good Boy/Assets/Scripts/GameFiles/LevelManager.cs]: This Awake() method is black magic to me. Why is this necessary? What does this do?

Comment (kocosephia) [reply WhosAGoodBoy/Who's A Good Boy/Assets/Scripts/GameFiles/LevelManager.cs]: Making the class a singleton. If level manager ever gets accidentally instantiated a second time, it will just destroy itself so it doesn't mess anything up.
